try{
  aaa
} catch (e) {
  // console.log(e);
  throw e;
}
